docs/source/llms/rag/notebooks/question-generation-retrieval-evaluation.ipynb

```diff
@@ -421,33 +299,85 @@
     "df.head(3)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "d6661fd5-8920-4801-a6af-388191b2a28e",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Save the results to the disk and load them back to avoid re-running the scraping process\n",
+    "# if you want to re-run the rest of the notebook with different parameters\n",
+    "df.to_csv(\"mlflow_docs_scraped.csv\", index=False)\n",
+    "df = pd.read_csv(\"mlflow_docs_scraped.csv\")"
```

Comment:
do you want to add a `.gitignore` file in this subdirectory for users who run this locally? that way if they choose to run this notebook it won't be potentially added to their commit

Reply:
I want to add this file to the mlflow repo so that people can reuse the files if they don't want to do it by themselves.